backend/pkg/config/cbor.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2022 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package config
+
+import "fmt"
+
+// Cbor represents the CBOR config.
+type Cbor struct {
+	Enabled bool `yaml:"enabled"`
+
+	// TopicNames is a list of topic names that shall be considered for cbor decoding.
+	// These names can be provided as regex string (e. g. "/.*/" or "/prefix-.*/") or as plain topic name
+	// such as "frontend-activities".
+	// This defaults to `/.*/`
+	TopicNames []string `yaml:"topicNames"`
```

Comment:
I believe we can use the type `RegexpOrLiteral` here. This is also done in the proto topic mappings and parses these strings automatically into a regex. This should help removing the cbor/service.go

Reply:
Ok

Reply:
Replaced with RegexpOrLiteral